modules/navigation.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] Kindly bear in mind that the use of a linter is a standard practice that ensures commitment to best practices in the real world in order to ensure ease of understandability, readability, maintainability, and scalability of code. In light of this, kindly resolve this linting issue without disabling it with this line :pray: